client/app/components/PopUp.js

```diff
@@ -25,13 +25,13 @@ class PopUp extends React.Component {
         NotificationManager.info(message);
         break;
       case 'success':
-        NotificationManager.success(message, 'Success', 1500);
+        NotificationManager.success(message, 'Success', 1300);
```

Comment:
http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

:P  (now I'm being picky)